pkg/util/json/json.go

```diff
@@ -442,6 +447,99 @@ func (j jsonObject) EncodeInvertedIndexKeys(b []byte) [][]byte {
 	return outKeys
 }
 
+// MakeJSONFromDecimal returns a JSON value given a apd.Decimal.
+func MakeJSONFromDecimal(v apd.Decimal) JSON {
+	return jsonNumber(v)
+}
+
+// MakeJSONFromArrayOfJSON returns a JSON value given a []JSON.
+func MakeJSONFromArrayOfJSON(v []JSON) JSON {
```

Comment:
I think you might have mentioned this before but I didn't understand at the time - I think you can get away without making this function if you add a check at the end of `MakeJSON` that just returns `JSON` implementors unchanged (#20217 also adds this functionality).